### ISSUE TITLE
immortal shoulders faceless void

### ISSUE THREAD

**Issue by anticore22** [ValveSoftware/Dota2-Gameplay#3850]
### Description

how the shoulders would look on a void with the effect of the color of the arcana style

### Screenshots

![76](https://user-images.githubusercontent.com/107674630/196034868-ec5ae0a8-de05-4144-8159-d8ab393d20e1.png)

**Comment by sbsolarski** [comment]
Good idea, the standard effect is especially jarring with the green version of shoulders.

**Comment by shaileshgarg** [comment]
#3443
